src/lib.rs

```diff
@@ -4,24 +4,28 @@ use crate::assets::LoadingPlugin;
 use crate::connect::ConnectMenuPlugin;
 use crate::login::LoginMenuPlugin;
 use crate::network::NetworkPlugin;
+use crate::rooms::RoomMenuPlugin;
 use crate::ui::BaseUiPlugin;
 
 mod assets;
 mod card;
 mod connect;
 mod login;
 mod network;
+mod rooms;
 mod ui;
 
 pub const EXPECTED_GAME: &str = "whist";
-pub const EXPECTED_VERSION: &str = "0.1.1";
+pub const EXPECTED_CORE_VERSION: &str = "0.1.1";
```

Comment:
Is that feasible in the long run. If we patch something which has no effect to the browser application?